### ISSUE TITLE
Missing LICENSE

### ISSUE THREAD

**Issue by stejoo** [cowops/debian-timesyncd#2]
I noticed you specify a "BSD" license in the README.md, but it is unclear to me which BSD license you are referring to (there are 4 of them). Adding a LICENSE file to the repository would be helpful in clearing that up.

Most popular choices among the BSD licenses are:
- [BSD 2-Clause “Simplified” License](<https://choosealicense.com/licenses/bsd-2-clause/>)
- [BSD 3-Clause “New” or “Revised” License](<https://choosealicense.com/licenses/bsd-3-clause/>)

I had a look at your other work/repositories and noticed you have used a MIT license in some of them. The MIT license would be similar to the BSD 2-Clause “Simplified” license.

tl;dr: Please add a LICENSE file.
